react-app/src/router.tsx

```diff
@@ -1,119 +1,138 @@
-import { createBrowserRouter } from "react-router";
-import * as F from "@/features";
 import * as C from "@/components";
+import { TimeoutModal } from "@/components";
+import * as F from "@/features";
 import {
   postSubmissionLoader,
   PostSubmissionWrapper,
 } from "@/features/forms/post-submission/post-submission-forms";
 import { QueryClient } from "@tanstack/react-query";
+import { createBrowserRouter } from "react-router";
 export const queryClient = new QueryClient();
-
 export const FAQ_TAB = "faq-tab";
 
+// These routes are divided into public and private, but this distinction
+// is not used for enforcing authentication. Instead, private routes include
+// a Timeout component that logs out a user after a period of inactivity.
+
+const PrivateWrapper = ({ element: Component }) => {
```

Comment:
Can we rename the routes to something else? Something like `TimedOutRoutes` or something along those lines?

Reply:
I guess my comment applies to anything affected by "private"